managers/asset-operations-manager.js

```diff
@@ -778,6 +778,16 @@ class AssetOperationsManager {
         };
     }
 
+    /**
+     * Wait for the finalization of an asset update operation.
+     * @async
+     * @param {string} UAL - The Universal Asset Locator of the asset.
+     * @param {Object} [options={}] - Optional parameters for waiting.
+     * @param {string} [options.blockchain] - The blockchain to monitor the update on.
```

Comment:
Just noticed we aren't validating frequency and maxNumberOfRetries - we should do that. You can probably add that easily and add this to the PR - add those validations to validateWaitAssetUpdateFinalization